tst/testspecial/run_gap.sh

```diff
@@ -14,7 +14,7 @@ gfile="$2"
 #    so the output is usable on other machines
 # 4) Set lower and upper memory limits, for consistency
 GAPROOT=$(cd ../..; pwd)
-( echo "LogTo(\"${outfile}.tmp\");" ; cat "$gfile" ; echo "QUIT;" ) |
+( echo "LogTo(\"${gfile}.tmp\");" ; cat "$gfile" ; echo "QUIT;" ) |
     "$gap" -r -A -b -m 256m -o 512m -x 800 2>/dev/null >/dev/null
-sed "s:${GAPROOT//:/\\:}:GAPROOT:g" < "${outfile}.tmp"
-rm "${outfile}.tmp"
+sed "s:${GAPROOT//:/\\:}:GAPROOT:g" < "${gfile}.tmp"
+rm "${gfile}.tmp"
```

Comment:
Why this change?

Reply:
Oh, this makes run_gap threadsafe (outfile isn't actually set to anything, so we just use the filename .tmp). I tend to run run_gap.sh in parallel.